### PR TITLE
Plenigo 978 wordpress default profile page

### PR DESCRIPTION
**Pull request by SebaSOFT**

### CODE REVIEW

File: plenigo_js/pl_settings.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+var plenigoSettings = plenigoSettings || {
```
Comment (maximilianschweitzer) [inline plenigo_js/pl_settings.js:1]: why is this here. Do we already have it?

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 2]
```diff
@@ -84,6 +121,9 @@ function plenigo_register_tinymce_js($plugin_array) {
         $plugin_array['plenigo_renew'] = plugins_url('../plenigo_js/tinymce-plenigo_renew-plugin.js', __file__);
         $plugin_array['plenigo_failed'] = plugins_url('../plenigo_js/tinymce-plenigo_failed-plugin.js', __file__);
         $plugin_array['plenigo_separator'] = plugins_url('../plenigo_js/tinymce-plenigo_separator-plugin.js', __file__);
+        // TODO add missing JS files
```
Comment (maximilianschweitzer) [inline plenigo_plugin/PlenigoShortcodeManager.php:124]: Where are these files? Is commenting out okay here?

File: plenigo_js/pl_settings.js (hunk 1 above)
Comment (SebaSOFT) [inline plenigo_js/pl_settings.js:1]: Non linked yet, it is part of the new editor. I needed to bring some stuff from the other branch and this got caught.

File: plenigo_plugin/PlenigoShortcodeManager.php (hunk 2 above)
Comment (SebaSOFT) [inline plenigo_plugin/PlenigoShortcodeManager.php:124]: Totally, in fact I have take them out in this branch but got re-added when you pulled master into this branch again.
